pennylane_lightning/core/src/simulators/lightning_gpu/StateVectorCudaManaged.hpp

```diff
@@ -212,12 +215,8 @@ class StateVectorCudaManaged
             index |= bit << (num_qubits - 1 - wires[k]);
         }
 
-        BaseType::getDataBuffer().zeroInit();
         const std::complex<PrecisionT> value(1.0, 0.0);
-        CFP_t value_cu = cuUtil::complexToCu<std::complex<Precision>>(value);
-        auto stream_id = BaseType::getDataBuffer().getDevTag().getStreamID();
-        setBasisState_CUDA(BaseType::getData(), value_cu, index, use_async,
-                           stream_id);
```

Comment:
Could you remove the `static_cast` from the line 214, please? It is no longer needed.

Reply:
Nice spot! Thanks